### ISSUE TITLE
credits page

### ISSUE THREAD

**Issue by peterstadler** [Edirom/MerMEId#55]
We should (and must?) list all contributors to the code base at some place. Probably best to put the names on a dedicated "Credits" page and link to it from the footer(?)

**Comment by peterstadler** [comment]
@daniel-jettka proposed to add something like 
```
<a href="https://github.com/Edirom/MerMEId/graphs/contributors">
  <img src="https://contrib.rocks/image?repo=Edirom/MerMEId" />
</a>
```
directly into the footer of the start page.

**Comment by daniel-jettka** [comment]
suggestion:
![image](https://user-images.githubusercontent.com/18614293/174227847-3e744c32-d3b7-4a7d-aec1-38450fae5f0f.png)

**Comment by daniel-jettka** [comment]
probably obsolete with #110 :-)

**Comment by daniel-jettka** [comment]
@peterstadler @AnastasiaWawilow 

Shall we still merge this or wait for #110 ?

**Comment by peterstadler** [comment]
IMHO this can be merged (when ready) without waiting for #110. It's already an improvement!
